docs/docs/coding/README.md

```diff
@@ -29,6 +29,27 @@ Person.decoder().decode(personJson)
 //sampleEnd
 ```
 
+## Enum Encoding/Decoding
+
+The `@json` annotation does not support `Enum` types, so a custom `Encoder` and `Decoder` must be used.
+You will be able to encode and decode using the following:
+
+```kotlin:ank:silent
```

Comment:
I would prefer as it was with two separate blocks:

For example giving the following `Enum`:
```kotlin:ank:silent
enum class Foo {
  A
}
```

You will be able to encode and decode using the following:

```kotlin:ank
import arrow.core.*
import helios.*
import helios.instances.*

//Encoding
val fooJson = Enum.Companion.encoder<Foo>().run {
  Foo.A.encode()
}

//Decoding
Enum.Companion.decoder<Foo>().decode(fooJson)
```